app/views/books/show.html.erb

```diff
@@ -10,6 +10,13 @@
   </ul>
 </section>
 <section id=reviews>
-  
+  <% @reviews.each do |review| %>
+  <ul id = <%="review_#{review.id}"%>>
```

Comment:
(14)I don't know if the syntax matters, but I thin these css tags have been using `-` instead of `_` ? Maybe it doesnt matter.